### DISCUSSION TITLE
Default Wacom tablet drivers not working with osu! lazer

### DISCUSSION THREAD

**Discussion by Alex-w3ps** [ppy/osu#25963] [Q&A]
Basically I'm trying to use my wacom tablet drivers and not use the built in ones on lazer because I prefer the smoothness of it however I can't move my cursor I can only click and then my cursor shows. It doesn't move with both dragging and hovering and I have both high precision mouse and the in game tablet disabled. I have tried fixes in similar discussions on this issue including tinkering with windows ink but even that did not help.

https://github.com/ppy/osu/assets/154455021/42383479-ead6-4d24-907f-bb4b3dac83d1

https://github.com/ppy/osu/assets/154455021/bbb8ce3c-3989-4d82-93a2-37ac7a70d449

**Comment by Wenderrrrr** [comment]
Try turning off windows ink in your pen settings
